packages/custom_lint_builder/lib/src/analyzer_plugin/plugin_client.dart

```diff
@@ -537,6 +568,10 @@ abstract class ClientPlugin {
   Future<Response?> _getResponse(Request request, int requestTime) async {
     ResponseResult? result;
     switch (request.method) {
+      case ForceReload.key:
+        await _debugHotReloader?.reloadCode();
```

Comment:
What is the purpose of manually triggering a "reloadCode"?

Since HotReloader is in automatic mode, won't it already reaload without having to do anything?
Considering the complexity of adding new events, I'm just wondering if we could remove `ForceReload` entirely

I get the value of pressing `r` to re-execute lints. I'm just not sure about this event

Reply:
As long as we re-report lints on both the linted file changes as well as the linter code changes, I think we can remove this. I probably added this just to have symmetry with flutter, except flutter doesn't listen to your code changes if you run it via the command line, so this is different. We can probably get rid of this.

Reply:
Nice, that's what I wanted to hear. I'll do it